app/presenters/subscription_presenter.rb

```diff
@@ -45,10 +44,10 @@ def search_criteria_field(field, value)
     { "#{field}": value }
   end
 
-  def render_location_filter(location_category, location, radius)
+  def render_location_filter(location, radius)
     return if location.empty?
 
-    return { location: I18n.t("subscriptions.location_category_text", location: location) } if location_category
+    return { location: I18n.t("subscriptions.location_category_text", location: location) } if location_polygon?(location)
```

Comment:
We could rename this translation, now.

Reply:
I think it's unnecessary to add a method for this conditional. You can just say `if (location && LocationPolygon.include?(location))` - and you could update the `LocationPolygon.include?` method so that it can cope with being passed a `nil` argument for `location`.